### PR TITLE
Use dark theme in settings window.

### PR DESCRIPTION
**Pull request by Sunxy88**
#545  Add 4 lines of code referred to [EditHabitActivity.kt](android/uhabits-android/src/main/java/org/isoron/uhabits/activities/habits/edit/EditHabitActivity.kt) to apply dark theme in settings window.

### CODE REVIEW

File: android/uhabits-android/src/main/java/org/isoron/uhabits/activities/settings/SettingsActivity.java [hunk 1]
```diff
@@ -23,17 +23,27 @@
 
 import org.isoron.androidbase.activities.*;
 import org.isoron.androidbase.utils.*;
+import org.isoron.uhabits.HabitsApplication;
```
Comment (iSoron) [inline android/uhabits-android/src/main/java/org/isoron/uhabits/activities/settings/SettingsActivity.java:26]: Please use wildcard imports

**Review by iSoron — CHANGES_REQUESTED** [review]
Thank you for the pull request, @Sunxy88. I have tested it and it works great. I only have one comment about code style. After that is fixed, I can merge it.

**Review by iSoron — APPROVED** [review]

**Comment by iSoron** [comment]
Looks great now. Thanks again, @Sunxy88!

**Comment by szpak** [comment]
Unfortunately, it doesn't work for me with 1.8.9 and 1.8.10 from Google Play. Dark theme enabled, but the settings panel is as white as it was before. I forced the app to restart. Android 11.

**Comment by iSoron** [comment]
@szpak This fix will become available in Loop 2.0

**Comment by szpak** [comment]
@iSoron Thanks for your explanation, I've just taken a brief look into default branch (dev), not spotting that 1.8.x is from master.

**Pull request merged**